src/app/auth.config.module.ts

```diff
@@ -33,9 +33,12 @@ import { environment } from '@env/environment';
           useRefreshToken: true,
           logLevel: LogLevel.Debug,
           customParamsAuthRequest: {
+            aud: environment.audience,
             client_secret: environment.clientSecret
           },
           customParamsCodeRequest: {
+            aud: environment.audience,
```

Comment:
I love the idea of an audience value as certain IDPs and auth flows require it. Do you know if this change works if the token issuer doesn't have an `aud` value returned? I'm wondering if it's better to make it an optional value, or maybe it doesn't really matter, but I want to avoid a scenario where `client_secret` is passed but `aud` is null or blank and the server returns errors due to malformed parameters